gadget/install/content.go

```diff
@@ -62,16 +62,35 @@ func makeFilesystem(params mkfsParams) error {
 	return udevTrigger(params.Device)
 }
 
+type mntfsParams struct {
```

Comment:
Normally i'd go for `Options` instead of `Params` - but there is precedence for using `Params` in this file, and I'd rather see consistency.